tests/unit/customizations/wizard/test_app.py

```diff
@@ -36,12 +31,482 @@
 from awscli.customizations.wizard.core import BaseStep, Executor
 
 
+@pytest.fixture
```

Comment:
How slow is all this stuff? I really don't want the unit tests to be significantly slowed down by these tests. It seems more reasonable for them to be functional tests.

Reply:
In the next commit, it should be much faster as there are not blanket sleep statements